### ISSUE TITLE
Arkos Tracker bin file

### ISSUE THREAD

**Issue by AlexandreMinard** [malbanGit/Vide#25]
I tried to play an Arkos tracker bin file.

When I call the init, strange things happen on the screen...
And trying to play will make the emulator very very very slow.

I'm wondering if I generate the bin file in this correct way.
Arkos Tracker asked me for an address when exporting.
What should i put?
Does that change something?

Regards,
Alexandre

**Comment by malbanGit** [comment]
Not enough information to help.
a) did you use arkos tracker version 1 (if you used version two - that
is not supported)
b) usually Vide can figure out what the address is - it also says what
it thinks in the "head" of the generated file
c) dunno - I can't "see" anything... and "strange things happen" doesn't
give me much of a clue...
Can you send me the project (zip file of the directory)?
vide AT malban.de

Am 20.05.2020 10:59 schrieb AlexandreMinard: 

> I tried to play an Arkos tracker bin file. 
> 
> When I call the init, strange things happen on the screen...
> And trying to play will make the emulator very very very slow. 
> 
> I'm wondering if I generate the bin file in this correct way.
> Arkos Tracker asked me for an address when exporting.
> What should i put?
> Does that change something? 
> 
> Regards,
> Alexandre 
> 
> --
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub [1], or unsubscribe [2].
 

Links:
------
[1] https://github.com/malbanGit/Vide/issues/25
[2]

**Comment by AlexandreMinard** [comment]
Yes I'm using version 1 of Arkos Tracker.
I exported to the address 00.

While preparing the project for you, I tried the bin file in an asm project: it worked fine!
Then on an empty 'C' project: it worked fine too.

On my project, I didn't see that my bank ram was full again...
After solving that, it's working now.

But, as I'm above the 30 000 cycles, the music is played slowly...

Thanks again!
